README.md

```diff
@@ -27,6 +27,13 @@ mkdir -p ~/.git-templates/hooks
 git config --global init.templatedir '~/.git-templates'
 ln -s $(pwd)/pre-commit/github_email_verify.sh ~/.git-templates/hooks/pre-commit
 ```
+
+#### Optional
+To add your github's account email as the default email to use, add the following line to your \*rc file (~/.bashrc, ~/.zshrc, etc)
```

Comment:
`In order to have your github's email account as the default email to be used, add the following line to your shell startup file (~/.bashrc, ~/.zshrc, etc.).`